ansible_collections/arista/avd/roles/eos_designs/templates/network_services/logic.j2

```diff
@@ -66,6 +74,12 @@
                                                                                                    map(attribute='structured_config') |
                                                                                                    first %}
 {%                                         endif %}
+{%                                         if bgp_svi_struct_cfg is not arista.avd.defined and svi_parent_profile.bgp.structured_config is arista.avd.defined %}
+{%                                             set bgp_svi_struct_cfg = svi_parent_profile.bgp.structured_config %}
+{%                                         endif %}
+{%                                         if bgp_svi_raw_eos_cli is not arista.avd.defined and svi_parent_profile.bgp.raw_eos_cli is arista.avd.defined %}
```

Comment:
raw_eos_cli does not require special treatment. It is just a string, so it will be merged automatically together with all other settings. It is _only_ structured_config we need to handle.

Reply:
What I meant is that the key `bgp.raw_eos_cli` will be merged as part of the combine in line 87, so please remove all these extra things for raw_eos_cli. You will get the most specific raw_eos_cli by just referencing svi.bgp.raw_eos_cli in the router-bgp-vlans template.